api/users/serializers.py

```diff
@@ -1,6 +1,30 @@
 from rest_framework import serializers as ser
-
 from api.base.serializers import JSONAPISerializer, LinksField, Link
+from website.models import User
+
+
+class ProfileDictSerializer(ser.ListField):
+    start_year = ser.CharField(allow_blank=True, allow_null=True)
+    start_month = ser.IntegerField(allow_null=True)
```

Comment:
I think using the DRF-provided date field makes the most sense here, and let the front-end handle the conversion.